doc/content/examples/example11b.md

```diff
@@ -0,0 +1,94 @@
+# Example 11b: Mesh refinement using `Marker` and `Indicator`, and demonstration of `SidesetMoment` and `AverageValue` postprocessors
+
+This example expands on [Example 11a](examples/example11.md) to demonstrate mesh refinement using `Markers` and `Indicators`. This example also demonstrates the use and verification of `SidesetMoment` and `AverageValue` postprocessors. For demonstration purposes,
+the beam model subjected to static loading from [Example 11a](examples/example11.md) is borrowed. This beam model is divided into three subdomains represented using three colors (red, green, and gray) in the figure below.
+
+!media media/examples/ex11b/ex11b_blocks.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:beam_blocks
+       caption=Beam model with the domain divided into three subdomains. The red, green, and gray colors represent three subdomains.
+
+## Mesh refinement using `Indicator` and `Marker`
+
+MASTODON is capable of automatic mesh refinement using the MOOSE's [Adaptivity System](syntax/Adaptivity/index.md). The [Adaptivity System](syntax/Adaptivity/index.md) can use an [`Indicator`](/Adaptivity/Indicators/index.md) to provide measurement of "errors" about the required mesh refinement. These "errors" are passed on to a [`Marker`](/Adaptivity/Markers/index.md) to decide which elements need refinement. Alternatively, the `Marker` can calculate the "errors" by itself to inform the mesh refinement without relying on the `Indicator`. Both these cases will be demonstrated. Specifically, the [`MinimumElementSizeMarker`](markers/MinimumElementSizeMarker.md) for mesh refinement is demonstrated. The [`MinimumElementSizeMarker`](markers/MinimumElementSizeMarker.md) is capable of refining elements based on either an `indicator` or an `element_size` value. These capabilities are first individually demonstrated below, and are then combined using the [`ComboMarker`](/markers/ComboMarker.md).
```

Comment:
Both "of" these cases will be demonstrated.

Reply:
Corrected